src/templates/template-common/main.py

```diff
@@ -1,19 +1,34 @@
 # main entrypoint
+#::: if ((it.argparser == 'fire')) { :::#
+def main(config_path, backend=None, **kwargs):
+    config_path = Path(config_path)
+    assert config_path.exists(), config_path
+    config = setup_config(config_path, backend, **kwargs)
+
+
+#::: } else if ((it.argparser == 'hydra')) { :::#
+@hydra.main(version_base=None, config_path=".", config_name="config")
```

Comment:
I do not understand how to pass backend to the config ?
Which backend is user by default ?

Reply:
By default `backend = None` but if we want to send a specific backend then `python main.py ++backend='gloo'`